### PR TITLE
feat(page.tsx): added top categories section to home page

### LINKED ISSUE #24: [FEAT]: ui/home - top course categories
### Summary
NA
### Answer following questions?
1. Is your feature related to FrontEnd? Yes
2. Is your feature related to Backend? No
3. Is your feature related to API Integration? No
### Resources
image

### PR DESCRIPTION
**Pull request by sub1120**
added top categories section to home page.

FYI.
- Test setup still not completed, got stuck.
- We can use div inside anchor tag, unless it contains any interactive element inside it(like another link, or buttom). Check out this reference (https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)

### CODE REVIEW

File: src/sampledata.ts [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+    name: "Label",
+    path: "courses/label",
+    iconId: "Cpu",
+    totalCourses: 63476,
   },
   {
-    name: "Mobile Development",
-    path: "courses/mobile-development",
+    name: "Buisness",
+    path: "courses/buisness",
```
Comment (shivamvijaywargi) [inline src/sampledata.ts]: Spelling mistake

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comment.

File: src/sampledata.ts (hunk 1 above)
Comment (sub1120) [reply src/sampledata.ts]: Corrected 👍🏻

**Review by shivamvijaywargi — APPROVED** [review]
All good

**Pull request merged**